src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingSlider/SliderMessages/StakingValidationMessage.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Button from '~shared/Button';
+import Numeral from '~shared/Numeral';
+import { getFormattedTokenValue } from '~utils/tokens';
+// import { TokenActivationContext } from '~users/TokenActivationProvider';
+
+import { useStakingWidgetContext } from '../../StakingWidgetProvider';
+import TokensLeftToActivate from './TokensLeftToActivate';
+
+import styles from './StakingValidationMessage.css';
+
+const displayName =
+  'common.ActionDetailsPage.DefaultMotion.StakingWidget.StakingValidationError';
+
+const MSG = defineMessages({
+  tokens: {
+    id: `${displayName}.tokens`,
+    defaultMessage: `Activate {leftToActivate} more {tokenSymbol} to be eligible to stake.`,
+  },
+  reputation: {
+    id: `${displayName}.reputation`,
+    defaultMessage: `The minimum reputation required to stake in this team is {minimumReputation} points. When this motion was created you only had {userReputation} points, so you can't stake on this motion. If you now have more than {minimumReputation} Reputation points, you will be able to stake on future motions.`,
+  },
+  stakeMoreTokens: {
+    id: `${displayName}.stakeMoreTokens`,
+    defaultMessage: `Oops! You don't have enough active tokens. To stake more than this, please activate more tokens.`,
+  },
+  stakeMoreReputation: {
+    id: `${displayName}.stakeMoreReputation`,
+    defaultMessage: `Oops! Your ability to stake is limited by the amount of Reputation you have. To be able to stake more on future motions, you'll need to earn more Reputation.`,
+  },
+  /*
+   * I think this copy is misleading. A motion can be staked more than the user's min stake (obviously).
+   * Shouldn't it be: Your minimum stake is more than the amount needed for the motion to proceed. A motion can't be staked over 100% of the amount needed.' ?
+   */
+  cantStakeMore: {
```

Comment:
I think this copy is misleading. A motion can be staked more than the user's min stake (obviously).
 Shouldn't it be: Your minimum stake is more than the amount needed for the motion to proceed. A motion can't be staked over 100% of the amount needed.' ?